Emby.Server.Implementations/Images/ArtistImageProvider.cs

```diff
@@ -0,0 +1,47 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Emby.Server.Implementations.Images;
+using MediaBrowser.Common.Configuration;
+using MediaBrowser.Controller.Drawing;
+using MediaBrowser.Controller.Dto;
+using MediaBrowser.Controller.Entities;
+using MediaBrowser.Controller.Entities.Audio;
+using MediaBrowser.Controller.Entities.Movies;
+using MediaBrowser.Controller.Entities.TV;
+using MediaBrowser.Controller.Library;
+using MediaBrowser.Controller.Playlists;
+using MediaBrowser.Controller.Providers;
+using MediaBrowser.Model.Entities;
+using MediaBrowser.Model.IO;
+using MediaBrowser.Model.Querying;
+
+namespace Emby.Server.Implementations.Images
+{
+    public class ArtistImageProvider : BaseDynamicImageProvider<MusicArtist>
```

Comment:
So this is just a non-functioning image provider? Why add it?

Reply:
It works perfectly, but these providers can't be disabled right now and some users might not want this. I figured I would disable it globally until we add a setting for these in the server configuration somewhere, it shouldn't be too difficult.